src/main/php/peer/stomp/Connection.class.php

```diff
@@ -193,7 +194,11 @@ public function sendFrame(Frame $frame) {
    *
    */
   protected function _connect(URL $url) {
-    $this->socket= new Socket($url->getHost(), $url->getPort(61612));
+    if ($url->getScheme() === 'stomp+ssl') {
+      $this->socket= new SSLSocket($url->getHost(), $url->getPort(61612));
+    } else {
+      $this->socket= new Socket($url->getHost(), $url->getPort(61612));
+    }
```

Comment:
This is not the correct backport - see https://github.com/xp-forge/stomp/compare/v11.0.0...v11.1.0